scout/server/blueprints/variant/verification_controllers.py

```diff
@@ -99,11 +103,15 @@ def variant_verification(
         for sample_obj in variant_obj["samples"]
     ]
     tx_changes = []
+    thermo_primer_link = ""
 
     if category == "snv":  # SNV
         view_type = "variant.variant"
         tx_changes = []
 
+        if case_obj["genome_build"] == 38:
+            thermo_primer_link = f"https://www.thermofisher.com/order/genome-database/searchResults?searchMode=keyword&CID=&ICID=&productTypeSelect=ceprimer&targetTypeSelect=ceprimer_all&alternateTargetTypeSelect=&alternateProductTypeSelect=&originalCount=0&species=Homo+sapiens&otherSpecies=&additionalFilter=ceprimer-human-exome&keyword=&sequenceInput=&selectedInputType=&chromosome={chromosome}&chromStart={position}&chromStop={position}&vcfUpload=&multiChromoSome=&batchText=&batchUpload=&sequenceUpload=&multiSequence=&multiSequenceNames=&priorSearchTerms=%28NR%29"
```

Comment:
Consider moving this to the links.py helpers!

Reply:
(and yeah, probably make it like external_primer_order_link and the target configurable.. 😅)

Reply:
Sorry for the delayed response! I blame the virus!

I've finally pushed some changes to this branch that moves the primer ordering link to config.py and the link contstruction to links.py. So now you can add a variable to config.py like this.

```
EXTERNAL_PRIMER_ORDER_LINK_38 = "https://www.primercompany.com/order?chr={chromosome}&pos={position}"
```

And a link will show up in the verification email for SNVs. `chromosome` and `position` are the only supported parameters at the moment. If the config variable is not set, things will look the same as before.

EXTERNAL_PRIMER_ORDER_LINK_37 is also supported for grch37. I don't know if it makes sense to have two different config variables or if it should generalized to something like this:

```
EXTERNAL_PRIMER_ORDER_LINK = [
    {"url":"http://some.url", genome_build=38},
    {"url":"http://another.url", genome_build=37}
]
```

Rather not change it unless you have a strong opinion. :)